CodeEdit/Features/Editor/TabBar/Tabs/Tab/EditorTabCloseButton.swift

```diff
@@ -104,17 +104,3 @@ struct EditorTabCloseButton: View {
         }
     }
 }
-
-#Preview {
-    @State var closeButtonGestureActive: Bool = false
-    @State var isHoveringClose: Bool = false
-
-    return EditorTabCloseButton(
-        isActive: false,
-        isHoveringTab: false,
-        isDragging: false,
-        closeAction: { print("Close tab") },
-        closeButtonGestureActive: $closeButtonGestureActive,
-        isHoveringClose: $isHoveringClose
-    )
-}
```

Comment:
Not sure it's super important, but since @Previewable is only available in macOS 14 we could mark the preview as such.
```suggestion

@available(macOS 14.0, *)
#Preview {
    @Previewable @State var closeButtonGestureActive: Bool = false
    @Previewable @State var isHoveringClose: Bool = false


    return EditorTabCloseButton(
        isActive: false,
        isHoveringTab: false,
        isDragging: false,
        closeAction: { print("Close tab") },
        closeButtonGestureActive: $closeButtonGestureActive,
        isHoveringClose: $isHoveringClose
    )
}
```

Reply:
Done

Reply:
Edit, removing this again because tests fail saying

> unknown attribute 'Previewable'

Reply:
Oh I wonder if that's a hint to what's happening with the tests failing? @Previewable should be available in the latest Xcode version

Reply:
Yeah, that thought crossed my mind. Xcode seems happy with it, it's just testing thats not. I wonder if we are somehow testing with a prior version. Do you know of any config around this that I am missing?

Reply:
Added back in, resolving.